.circleci/bin/test-airflow-image.py

```diff
@@ -287,9 +287,9 @@ def test_airflow_configs(scheduler, docker_client):
 
         assert scheduler.check_output(
             f"cat {config_file_path} | "
-            "grep '^auth_backend' | awk '{print $3}'"
+            "grep '^auth_backends' | awk '{print $3}'"
```

Comment:
```suggestion
            "grep '^auth_backend' | awk '{print $3}'"
```

We will have to keep this `auth_backend` as <2.3.0 versions will still use it and since is just grep it should work fine for 2.3.0+ too